workflows/offline_ml_diags/offline_ml_diags/create_report.py

```diff
@@ -98,12 +100,50 @@ def render_model_sensitivity(figures_dir, output_dir) -> str:
     )
 
 
+def render_time_mean_maps(
+    figures_dir, output_dir, ds_diags, column_integrated_vars
+) -> str:
+    report_sections: MutableMapping[str, Sequence[str]] = {}
+
+    # time averaged column integrated quantity maps
+    for var in column_integrated_vars:
+        ds_diags[f"error_in_{var}"] = (
```

Comment:
To avoid having to manage a case where `derivation_plot_coords=None` and the ambiguity about faceting in the subsequent plotting function, you could instead assign a coord `derivation=['error']` to the `var` DataArray for this case. Up to you but might be simpler.